src/Protocol/Protocol.php

```diff
@@ -212,15 +212,19 @@ public function getAckFrame(Frame $frame, $transactionId = null)
      *
      * @param \Stomp\Transport\Frame $frame
      * @param string $transactionId
+     * @param bool $requeue
      * @return \Stomp\Transport\Frame
      * @throws StompException
      */
-    public function getNackFrame(Frame $frame, $transactionId = null)
+    public function getNackFrame(Frame $frame, $transactionId = null, $requeue = null)
     {
         if ($this->version === Version::VERSION_1_0) {
             throw new StompException('Stomp Version 1.0 has no support for NACK Frames.');
         }
         $nack = $this->createFrame('NACK');
+        if ($requeue === false) {
```

Comment:
It should be possible to send `requeue=>true` too, when I read http://www.rabbitmq.com/amqp-0-9-1-reference.html#basic.nack.requeue it feels as if there is an way to configure the server to change the default behavior, not sure if this applies to the stomp plugin http://www.rabbitmq.com/stomp.html too.
But from my point of view it makes sense to allow true and false to be transmitted.
So the condition should be based on `$requeue !== null`.